src/application/extension-popup/popup-home-view/components/section-tile.tsx

```diff
@@ -18,26 +20,28 @@ export const SectionTile = ({
   subtitle,
   href,
   disabled,
+  iconContainerClassName,
 }: SectionTileProperties) => {
   return (
     <a
       href={href}
+      target={href ? "_blank" : undefined}
+      rel={href ? "noopener noreferrer" : undefined}
       className={classes(
         'relative rounded-lg p-4 text-center shadow-md',
         !disabled &&
           'transition-transform duration-[600ms] hover:scale-105 hover:bg-gray-50',
         disabled && 'cursor-not-allowed bg-gray-200 opacity-50',
       )}
     >
-      <div className="mb-2 flex justify-center space-x-3">
+      <div className={classes('mb-2 flex justify-center space-x-3', iconContainerClassName)}>
         {icons.map((icon) => {
           return (
             <img
               key={icon.iconSrc}
               src={icon.iconSrc}
               alt={icon.iconAlt}
-              className="size-8 rounded-lg"
-            />
+              className={`${icon.className ? icon.className : 'w-8 h-8'} rounded-lg`}            />
```

Comment:
Can we use classes function here? I believe this function solves some tailwind problems with conditional classNames.
```className={classes('h-8 w-8 rounded-lg', icon.className)}```